draft-ietf-oauth-v2-1.md

```diff
@@ -1582,250 +1722,17 @@ only):
 
 The authorization server MUST authenticate the client.
 
+## Refresh Token Grant {#refreshing-an-access-token}
```

Comment:
This feels abrupt jumping right into the extension section. This could use a sentence or two here re-introducing what the refresh token grant is.

Reply:
better?